homeassistant/components/logbook.py

```diff
@@ -286,6 +288,29 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data.get(ATTR_ENTITY_ID)
+                dsp_name = data.get(ATTR_DISPLAY_NAME) or entity_id
```

Comment:
Actually, instead of `data.get` just use `data[ATTR_ENTITY_ID]` and so on, since they are all passed in the event_data every time.

Reply:
My reasoning for this was if the event is sent from something else or so and not from call_service itself, hence did it this way from a safety perspective.
If you do not believe this is required and we assume that data is always there then I can process it accordingly. :-)

Reply:
Since this is a HomeKit only event, I think we'll be fine. We can even streamline it further and integrate those calls directly into the message construction (like Alexa is doing). No need to define a new variable for just one interaction. An exception for me would be the `entity_id` (if we want to be a crazy, it could even remain with `get` ;)

Reply:
All done. Did not go crazy, too early in the morning here right now. :-)